build.sbt

```diff
@@ -196,6 +202,20 @@ lazy val `core-common` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     )
   )
 
+lazy val `core-logs` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+  .crossType(CrossType.Pure)
+  .in(file("core/logs"))
+  .enablePlugins(NoPublishPlugin)
```

Comment:
I think it's a safe start. We would still be able to publish a patch or minor release.